lightning/src/onion_message/messenger.rs

```diff
@@ -1047,18 +1077,35 @@ where
 	/// enqueueing any response for sending.
 	pub fn handle_onion_message_response<T: OnionMessageContents>(
 		&self, response: ResponseInstruction<T>
-	) {
-		if let ResponseInstruction::WithoutReplyPath(response) = response {
-			let message_type = response.message.msg_type();
-			let _ = self.find_path_and_enqueue_onion_message(
-				response.message, Destination::BlindedPath(response.reply_path), None,
-				format_args!(
-					"when responding with {} to an onion message with path_id {:02x?}",
-					message_type,
-					response.path_id
-				)
-			);
-		}
+	) -> Result<Option<SendSuccess>, SendError> {
+		let (response, create_reply_path) = match response {
+			ResponseInstruction::WithReplyPath(response) => (response, true),
+			ResponseInstruction::WithoutReplyPath(response) => (response, false),
+			ResponseInstruction::NoResponse => return Ok(None),
+		};
+		let logger = WithContext::from(&self.logger, None, None, None);
```

Comment:
Don't think we need this if we aren't adding any context.

Reply:
Makes sense! Using a simple self.logger will be a better fit for this part of the code!
Addressed in [57bca64](https://github.com/lightningdevkit/rust-lightning/commit/57bca64bc1b7b3bbfc34fffa395cf9a38d5f8d48)
Thanks for the pointer!